apis/r/tests/testthat/test-Factory.R

```diff
@@ -1,20 +1,35 @@
 test_that("DataFrame Factory", {
     uri <- tempfile()
 
-    # check that straight use of new() errors, but 'with handshake' passes
+    # Check that straight use of new() errors, but 'with handshake' passes
     expect_error(SOMADataFrame$new(uri))
     expect_silent(d1 <- SOMADataFrame$new(uri, internal_use_only = "allowed_use"))
```

Comment:
Not part of this change but I like the way that the "secret handshake" pattern has made its way over to R here.